### ISSUE TITLE
Net::HTTPNotFound error when bundling with rails-assets-jquery-colorbox

### ISSUE THREAD

**Issue by sentience** [tenex/rails-assets#112]
I have this in my `Gemfile`:

```
gem 'rails-assets-jquery-colorbox'
```

and this in my `Gemfile.lock`:

```
rails-assets-jquery-colorbox (1.4.36)
```

As of today, however, when I `bundle install`, it fails with this:

```
Fetching source index from https://rubygems.org/
Fetching source index from https://rails-assets.org/
Net::HTTPNotFound: File not found:
/quick/Marshal.4.8/rails-assets-jquery-colorbox-1.4.36.gemspec.rz
```

The file in question does indeed appear to be missing: http://rails-assets.org/quick/Marshal.4.8/rails-assets-jquery-colorbox-1.4.36.gemspec.rz

Version 1.4.36 of jquery-colorbox is certainly listed in the official [component list](https://rails-assets.org/components). What could be causing this?

**Comment by sentience** [comment]
That file is being served now. Did someone fix it, or is this an intermittent issue?

**Comment by sheerun** [comment]
There is slight delay (few minutes) before requesting new component and time it's builded. Maybe that was the reason.

**Comment by sentience** [comment]
Hmm I don’t think so. That particular gem version had been in our Gemfile.lock and installed successfully several times over the past several days.

**Comment by sheerun** [comment]
Well, that's interesting. I guess it's because of reindexing. Thank you for reporting this.

**Comment by sheerun** [comment]
I just discovered our rake reindex first removes all previous gemspec.rz files, before performing new reindex, and in our case it takes around 5 minutes because of the number of gems...

This is probably rubygems issue, but I guess we can mitigate it somehow manually moving index files when index is generated. Ping @teamon

**Comment by teamon** [comment]
How about reindexing into some tmp dir and then just rm && mv?

**Comment by sheerun** [comment]
Closed by d571380f5ea

**Comment by sentience** [comment]
:thumbsup:
